aspnetcore/blazor/fundamentals/routing.md

```diff
@@ -135,30 +135,37 @@ Arbitrary items are supported as content of the `<NotFound>` tags, such as other
 
 ## Route to components from multiple assemblies
```

Comment:
Can we cover both `Router.AdditionalAssemblies` and `MapRazorComponents<TComponent>().AddAdditionalAssemblies()` in this section? The former is used when the Blazor `Router` is handling page navigations interactively. The latter is used when routing requests to components on the server using endpoint routing.

Reply:
I moved the coverage over to this section and left a cross-link to this content in the *Render Modes* article.